contributors.py

```diff
@@ -72,7 +73,16 @@ def main():
         sponsor_info,
         link_to_profile,
     )
-    # write_to_json(contributors)
+    json_writer.write_to_json(
+        filename="contributors.json",
+        start_date=start_date,
+        end_date=end_date,
+        organization=organization,
+        repository_list=repository_list,
+        sponsor_info=sponsor_info,
+        link_to_profile=link_to_profile,
+        contributors=contributors,
```

Comment:
I think contributors is the first argument, right?  first argument to method but last on output.  Do we want to make same?

Reply:
ah, doesn't matter.  You're calling out the argument names.  NM

Reply:
Yeah I like to do this for readability and so I don't have to remember the order when there are so many args. Hopefully that's not too confusing.